src/riscv-integration.adoc

```diff
@@ -764,10 +764,14 @@ Instruction access fault
 
 | .>|2 +
 0 +
+*{cheri_excep_mcause}* +
+*{cheri_excep_mcause}* +
 8,9,11 +
 3 +
 3 .<|Illegal instruction +
 Instruction address misaligned +
+*CHERI <<ASR-permission>> fault* +
+*CHERI branch/jump target out of bounds fault* +
```

Comment:
I agree that the ASR permission might be missing, but the jump/branch failures are already below. Note that a few rows below it says:

> *Prior to address translation for an explicit memory access or jump:*
*CHERI fault*